scripts/monorepo/fix-dev-version.js

```diff
@@ -0,0 +1,5 @@
+const fsExtra = require('fs-extra');
+const packageJsonPath = 'packages/compass/package.json';
+const packageJson = fsExtra.readJSONSync();
```

Comment:
If I get it right, this probably should have a path

```suggestion
const packageJson = fsExtra.readJSONSync(packageJsonPath);
```